assets/css/setup-checklist.css

```diff
@@ -65,7 +93,9 @@
 	-webkit-box-direction: normal;
 	-ms-flex-direction: column;
 	flex-direction: column;
-	max-width: 40%;
+	-webkit-box-align: stretch;
+	-ms-flex-align: stretch;
+	align-items: stretch;
```

Comment:
These lines are present in the Calypso styles. I changed it slightly when the toggle was not present.